src/components/icon/icon.service.ts

```diff
@@ -8,6 +8,9 @@ import { uxIconset } from './iconsets/ux-iconset';
 
 @Injectable()
 export class IconService {
+    readonly options = inject<IconModuleOptions>(ICON_OPTIONS_TOKEN, { optional: true });
```

Comment:
In this case we shouldn't need the generic here:

```suggestion
    readonly options = inject(ICON_OPTIONS_TOKEN, { optional: true });
```

The correct approach would be to update the token to specify the type:

```ts
export const ICON_OPTIONS_TOKEN = new InjectionToken<IconModuleOptions>('ICON_OPTIONS_TOKEN');
```

This should be done for all our tokens.